core/src/main/scala/cats/syntax/monadError.scala

```diff
@@ -0,0 +1,16 @@
+package cats
+package syntax
+
+trait MonadErrorSyntax {
+
+  implicit def monadErrorSyntax[F[_], E, A](fa: F[A])(implicit F: MonadError[F, E]): MonadErrorOps[F, E, A] =
+    new MonadErrorOps(fa)
```

Comment:
No, I think that you are right. Coincidentally this just came up in a [gitter conversation](https://gitter.im/typelevel/cats?at=573b10d2e2996a5a42c91a12).